module/VuFind/src/VuFind/RateLimiter/RateLimiterManagerFactory.php

```diff
@@ -41,6 +41,7 @@
 use Symfony\Component\RateLimiter\Storage\CacheStorage;
 use Symfony\Component\RateLimiter\Storage\StorageInterface;
 use VuFind\RateLimiter\Storage\CredisStorage;
+use VuFind\RateLimiter\Turnstile\RejectAll;
```

Comment:
I wasn't sure what package to put this in.  Theoretically it's not Turnstile-specific so should be in RateLimiter or perhaps a new RateLimiter\Policy package.  But Turnstile was my primary use case so I put it here for now.

Reply:
I defer to @EreMaijala, but since this class implements LimiterInterface and is not Turnstile-specific, maybe VuFind\RateLimiter\Limiter would be a logical namespace. Or maybe just in the top-level VuFind\RateLimiter namespace, which would parallel the way Symfony places LimiterInterface within their own package.

Reply:
I wouldn't mind putting it in the top-level VuFind\RateLimiter namespace. It's not crowded and mirroring Smyfony's layout makes sense to me.